lib/chromic_pdf/template.ex

```diff
@@ -17,14 +25,20 @@ defmodule ChromicPDF.Template do
           | {:header, blob()}
           | {:footer, blob()}
 
+  @type paper_size :: :a4 | :letter | {float(), float()}
+
   @type style_option ::
-          {:width, binary()}
-          | {:height, binary()}
+          {:size, paper_size()}
           | {:header_height, binary()}
           | {:header_font_size, binary()}
           | {:footer_height, binary()}
           | {:footer_font_size, binary()}
 
+  @paper_sizes_in_inch %{
+    a4: {8.3, 11.7},
+    letter: {8.5, 11.0}
```

Comment:
it letter clear enough? Should this maybe be `us_letter` ?

Reply:
I think from the context it should be clear, but I changed it nonetheless.

From [Wikipedia: en/Paper_size](https://en.wikipedia.org/wiki/Paper_size)

> By 1977, A4 was the standard letter format in 88 of 148 countries. Today the standard has been adopted by all countries in the world except the United States and Canada. In Mexico, Costa Rica, Colombia, Venezuela, Chile, and the Philippines, the US letter format is still in common use, despite their official adoption of the ISO standard.